home/ivp-planning/advanced-tools/dxf-tool.md

```diff
@@ -0,0 +1,80 @@
+# DXF Files
+
+In iVP it is possible to import and export DXF files, that function as a simplified 2d visualisation of 3d models. They can be useful for illustrating a planned layout.
+
+After a DXF file is imported, it will be added to the [hierarchy](../user-interface/the-machine-list.md) and can be [edited](../machines/selecting-and-moving-objects.md) as any other object.
+
+{% hint style="warning" %}
+This is different to an [imported PDF](../getting-started/importing-pdfs.md), which will not be moveable in iVP but can be used as a overlay for the [floor plan](../user-interface/the-floor-plan.md).
+{% endhint %}
+
+{% hint style="info" %}
+For better understanding and training we created a [<img src="../../../.gitbook/assets/YouTube_icon.png" alt="" data-size="line"> YouTube Tutorial](https://www.youtube.com/watch?v=Maxivv824Dk&list=PLlzoGkRUR67houzn5F5ejD3R-kQrDcps5&index=18), where you can find a detailed explanation how to import and export of DXF files.
+{% endhint %}  // Video TBD
+
+## Importing a DXF File:
+
+To import a DXF click **File -> DXF -> Import DXF**. 
+
+![](../../../.gitbook/assets/DXF\_Import\_open.jpg)
+
+A new window will open that allows to navigate to the folder containing the DXF file to be imported. to open the file, simply double click it **or** select it and click Open.
+
+![](../../../.gitbook/assets/DXF\_Import\_file.jpg)
+
+Before iVP Planning imports a DXF file, a prompt will ask to set some parameters that specify how the DXF file will be imported. 
+
+![](../../../.gitbook/assets/DXF\_Import\_settings.jpg)
+
+* _**File path:**_ this line represents the location of the to be imported file, change it by clicking the three dots
+* _**As references:**_ this toggle defines whether or not you reference the dxf file. If checked, you will need to keep the dxf file accessible. (!changing the name or location of that file will break the reference!) 
```

Comment:
Remove the leading exclamation mark in the parenthesis

Reply:
removed